src/EthereumVaultConnector.sol

```diff
@@ -258,6 +264,23 @@ contract EthereumVaultConnector is Events, Errors, TransientStorage, IEVC {
         return isAccountOperatorAuthorizedInternal(account, operator);
     }
 
+    /// @inheritdoc IEVC
+    function setLockdownMode(bytes19 addressPrefix, bool enabled) public payable virtual onlyOwner(addressPrefix) {
+        if (ownerLookup[addressPrefix].isLockdownMode == enabled) {
+            revert EVC_InvalidLockdownModeStatus();
```

Comment:
What about simply exiting? A panicked user might hit the button multiple times and be spooked even more when things revert. (nitpick)

Reply:
again, consistency. the operator-related functions work similarly. but I think I agree, it's better to have a no-op in this case. thanks!